### PR TITLE
Prepared new release

### PR DESCRIPTION
**Pull request by Photon89**
Updated the bug reporting URL. Still to replace:

- https://translations.launchpad.net/shutter requires to set up a new localization service
- https://answers.launchpad.net/shutter probably should be removed since GitHub has no suiting replacement (?)
- https://launchpad.net/shutter/+download can be replaced once a first release tarball is ready

### CODE REVIEW

**Review by vadi2 — APPROVED** [review]

**Comment by DarthGandalf** [comment]
> https://answers.launchpad.net/shutter 

* Keep it on launchpad?
* Use one of Stack Exchange sites? https://askubuntu.com/questions/tagged/shutter

**Comment by Photon89** [comment]
I like the askubuntu idea a lot, there is an active community there as opposed to the Launchpad answers page for Shutter!

**Comment by vadi2** [comment]
Agree, askubuntu is better.

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+* updated translations
+
+* removed dependencies:
+
+    --  Arch: gnome-perl, gnome-vfs-perl
+    --  Debian: libgnome2-perl, libgnome2-vfs-perl
+
+* new dependencies:
+
+    --  Arch: perl-number-bytes-human, perl-glib-object-introspection
```
Comment (DarthGandalf) [inline CHANGES:13]: I don't know arch, but is perl prepended or appended?

**Review by DarthGandalf — APPROVED** [review]
Also README (both of them... why there are two?) mentions launchpad.

**Comment by DarthGandalf** [comment]
https://shutter-project.org/contact/ also mentions launchpad, and more.

File: CHANGES (hunk 1 above)
Comment (Photon89) [reply CHANGES:13]: That's fine, I actually maintain the Arch package, here is the AUR page with the full dependency list: https://aur.archlinux.org/packages/shutter-git/ I will have a look at the README and website, thanks for pointing that out!

**Comment by Photon89** [comment]
Got another idea: What about adding a "question" tag to the Github issues and let what was Launchpad answers run over Github issues? It is solved this way for Telegram: https://github.com/telegramdesktop/tdesktop/issues If you create a new issue, you can select if it should be a bug report, a feature request or a question, the issue then gets a corresponding tag. I don't know how they configured Github this way though, possibly it is some kind of paid feature.

**Comment by DarthGandalf** [comment]
Oh, interesting. I wasn't aware of that feature.

https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

**Comment by Photon89** [comment]
Oh, that's great reading, I will play around with it in a test repo and try to set it up here then, if there are no objections!

**Comment by Photon89** [comment]
What do you think: https://github.com/Photon89/test/issues/new/choose

**Comment by DarthGandalf** [comment]
Looks fine

**Comment by vadi2** [comment]
The chooser is good, but hardly anybody will be interested in filling out such detailed forms. It needs to be simplified by a lot.

**Comment by Photon89** [comment]
@vadi2 To be honest, I just started with the default and adapted it a bit for Shutter. Please feel free to edit the template at https://github.com/shutter-project/shutter/issues/templates/edit !

**Comment by vadi2** [comment]
Sure done.

**Comment by Photon89** [comment]
Yeah, this definitely looks better now!

**Comment by DarthGandalf** [comment]
Hi, any progress on this?

**Comment by vadi2** [comment]
Hmm what is there left to do?

**Comment by Photon89** [comment]
I think, we wanted to get the new website running with information about the move to Github. And then I made a (bad) proposal for the main page and somehow got busy and didn't make a better one, see https://github.com/shutter-project/shutter-jekyll-site/pull/13 ... Sorry for that!

Also there was an issue that the latest git commit fails to open image files which, I think, was fixed by Alexey, but now I just retested it and it again hit me. Need to check what is going on there.

**Comment by Photon89** [comment]
Regarding the regression when opening an image file with Shutter, I found another occasion when it happens and reopened the corresponding report: https://github.com/shutter-project/shutter/issues/291#issuecomment-721214061

**Comment by DarthGandalf** [comment]
Any other blocker? :)

**Comment by Photon89** [comment]
I think, I am the only blocker right now... Sorry, guys, I got quite busy with my job... Please feel free to do whatever needs to be done to get the new website and release online if you have the capacity! If you need me to post a news on Launchpad, let me know!

**Pull request merged**